test/to-port-to-wpts/level2/style.js

```diff
@@ -352,4 +352,45 @@ describe("level2/style", () => {
     assert.equal(document.body.style.width, "auto");
     assert.equal(document.body.style.cssText, "width: auto;");
   });
+  specify("ensure cssom uses the correct jsdom instance's globalObject", () => {
```

Comment:
Create new tests that cannot be WPTs in https://github.com/jsdom/jsdom/blob/main/test/api/from-outside.js